installer_script.nsi

```diff
@@ -0,0 +1,84 @@
+# NSIS script to create an installer. command:  makensis.exe /LAUNCH .\installer_script.nsi
+
+# Include Modern UI
+!include "MUI2.nsh"
+
+# Compression method, '/SOLID lzma' takes least space
+# setCompressor /SOLID lzma
+
+!define MUI_ICON "src\wwwroot\favicon.ico"
+!define MUI_UNICON "src\wwwroot\favicon.ico"
+!define UNINSTALLER "uninstaller.exe"
+!define REG_UNINSTALL "Software\Microsoft\Windows\CurrentVersion\Uninstall\StableSwarmUI"
+
+# Define variables
+Name "StableSwarmUI"
+OutFile "StableSwarmUI-Installer.exe"
+# Var FilePath
+
+# Default installation directory
+InstallDir "$PROFILE\StableSwarmUI"
+
+# Request application privileges for Windows Vista and later
+RequestExecutionLevel admin
+
+# Pages
+!insertmacro MUI_PAGE_WELCOME
+!insertmacro MUI_PAGE_LICENSE "LICENSE.txt"
+!insertmacro MUI_PAGE_COMPONENTS
+!insertmacro MUI_PAGE_DIRECTORY
+!insertmacro MUI_PAGE_INSTFILES
+!insertmacro MUI_PAGE_FINISH
+
+!insertmacro MUI_UNPAGE_WELCOME
+!insertmacro MUI_UNPAGE_CONFIRM
+!insertmacro MUI_UNPAGE_INSTFILES
+!insertmacro MUI_UNPAGE_FINISH
+
+!insertmacro MUI_LANGUAGE "English" # The first language is the default language
+
+# Sections
+Section "StableSwarmUI (required)"
+    SetOutPath "$INSTDIR"
+    # Add files to be installed
+    File /r /x *.bat /x *.sh /x *.ps1 /x StableSwarmUI-Installer.exe /x DOCKERFILE /x .dockerignore \
+    /x docker-compose.yml /x colab /x .github /x .git /x bin ${__FILEDIR__}\*.*
```

Comment:
You're just... installing the Swarm files to a path? So still a raw source file install, but minus the scripts and files essential to actually run it

Reply:
I figured since the source files are packaged into an exe now they would not need the other scripts. In the powershell script the source code is built so they could show up as an application since I thought the batch file would be less familiar for a normal consumer. I can leave the mac and linux bash files but I thought it wouldn't be too helpful on a windows installer.

Reply:
But it's still including eg `*.cs` code files is what's confusing me - it's still dumping source code, just minus the bits to run it.

It also appears to not be excluding eg `Data`, `Models`, `dlbackend`, etc. that would make a -giant- installer file if ever accidentally ran from an in-use directory

Reply:
Sorry, I seemed to overlooked that. I'll remove the .cs files and have it exclude the mentioned directories but still create empty folders for the models and similar folders. I thought the script to make the installer would only be ran by the remote repository for the newest build but I suppose it doesn't hurt to make it foolproof if ran on a computer with a model loaded.

Reply:
Swarm's code autogenerates all needed folders already you don't need to worry about creating them.